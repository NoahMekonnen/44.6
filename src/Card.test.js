import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

it("it renders", function(){
  render(<Card/>)
})

it("should match snapshot", function(){
    const { asFragment } = render(<Card caption="sea" src="image1.png" currNum={1} totalNum={3}/>)
    expect(asFragment()).toMatchSnapshot()
})