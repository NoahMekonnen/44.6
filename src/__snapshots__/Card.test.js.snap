// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      sea
    </h4>
    <img
      alt="sea"
      class="Card-image"
      src="image1.png"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
